spec/features/welcome/index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "Welcome Index Page", type: :feature do
+  it "displays area to search parks by state", :vcr do
+    visit root_path
+
+    select "Tennessee"
+    click_button "Find Parks"
+
+    expect(current_path).to eq(parks_path)
+    expect(page.status_code).to eq(200)
+    expect(page).to have_content(ParksFacade.total_parks_found("TN"))
+    expect(ParksFacade.total_parks_found("TN").to_i).to eq(15)
```

Comment:
This type of assertion would go in the Facade unit test but I see what you were thinking here. See comment in the facade about returning both the parks and the total in a collection as I think this will help you be able to display total in the view!